lib/routes/apnews/mobile-api.ts

```diff
@@ -0,0 +1,94 @@
+import { Route, ViewType } from '@/types';
+import { asyncPoolAll, fetchArticle } from './utils';
+import ofetch from '@/utils/ofetch';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/mobile/:path{.+}?',
+    categories: ['traditional-media'],
+    example: '/apnews/mobile/ap-top-news',
+    view: ViewType.Articles,
+    parameters: {
+        tags: {
+            description: '',
+            options: [{ value: '', label: '' }],
+            default: 'ap-top-news',
+        },
+    },
```

Comment:
Missing parameter description for https://github.com/DIYgod/RSSHub/blob/cbeb69ca67b134e1d2671e518a524f0f1bcc263c/lib/routes/apnews/mobile-api.ts#L7